docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -43,16 +43,18 @@ mandatory: True
 id: Interview Order
 code: |
   template_upload
+  get_all_fields
   # Run pdf_field_tester.yml if form is a pdf.  
-  if template_upload[0].filename.endswith('pdf'):    
+  if template_upload[0].filename.endswith('pdf'):
+    if len(get_person_variables(all_fields, custom_only=True)) > 0:
```

Comment:
This check doesn't seem to work correctly, at least not with this PDF
[NL_small_claims_repaired(1).pdf](https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/files/6542558/NL_small_claims_repaired.1.pdf)

Reply:
That should be resolved with the new way of showing possible custom people.